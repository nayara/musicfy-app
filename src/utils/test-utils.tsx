import { RenderOptions, render } from "@testing-library/react";
import { FC, ReactElement, ReactNode } from "react";
import { Providers } from "src/providers";

const AllTheProviders: FC<{ children: ReactNode }> = ({ children }) => {
  return <Providers>{children}</Providers>;
};

const customRender = (
  ui: ReactElement,
  options?: Omit<RenderOptions, "wrapper">
) => render(ui, { wrapper: AllTheProviders, ...options });

export * from "@testing-library/react";
export { customRender as render };
