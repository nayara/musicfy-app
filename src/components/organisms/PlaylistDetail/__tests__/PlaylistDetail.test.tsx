import { render } from "../../../../utils/test-utils";
import PlalistDetail from "..";

describe("<PlalistDetail />", () => {
  it("should render playlist correctly", () => {
    const { container } = render(<PlalistDetail />);

    expect(container).toMatchSnapshot();
  });
});
