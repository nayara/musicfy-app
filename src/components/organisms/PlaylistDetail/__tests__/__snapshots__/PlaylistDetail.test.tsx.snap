// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlalistDetail /> should render playlist correctly 1`] = `<div />`;
